### DISCUSSION TITLE
Installed Lazyvim Starter on a fresh MacBook. set up and tried to open lazygit, getting error

### DISCUSSION THREAD

**Discussion by escapingSamsara** [LazyVim/LazyVim#4340] [General]
I was installing neovim and lazyvim starter on a new macbook tonight and when i tried to open lazygit within lazyvim (leader gg), i got greeted by this error (see image). 
![lazyvim-distrubtion-gives-me-an-error-when-trying-to-open-lazygit](https://github.com/user-attachments/assets/9397d74d-bd20-40ca-8393-07c0d7016968)

When i run lazygit (which i manually installed after this with brew) in the same project from within the terminal it opens and works as intendend, but not from within lazyvim (having the project open). how can i fix this ?

**Comment by Jomik** [comment]
This seems like `lazygit` is accessible on your PATH inside neovim. How are you opening neovim? Are you using the same terminal
